sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece.processing.service.ts

```diff
@@ -5,7 +5,7 @@ import {
   ConfigService,
   InstitutionIntegrationConfig,
 } from "@sims/utilities/config";
-import { getFileNameAsCurrentTimestamp } from "@sims/utilities";
+import { getFileNameAsCustomizedCurrentTimestamp } from "@sims/utilities";
```

Comment:
We have almost the same method in place, can we reuse it?

in date-utils.ts in lib

![image](https://github.com/bcgov/SIMS/assets/54600590/7a917606-6906-43ce-a1df-8b6b1b76f381)

Reply:
The format we use is different, initially i created by passing the format as a parameter to the method. But eventually I had to touch in places my changes are not required, so created a new method with this approach.

Reply:
We can have the existing timestamp in the method as default format as it has been used in multiple places and give an option to use custom format as well

Reply:
I believe we can have it that way customized and pass the particular format in all the places by declaring them in a global constant. Is this change a blocker to be move forward? I can make the change, please advice @andrewsignori-aot @andrepestana-aot @lewischen-aot @sh16011993

Reply:
For such methods, I would be ok either way.
We can also allow the first method to receive the optional format and have the second one calling the first. In this case, the second one would be just a shortcut for the first one.
Not a blocker for this scenario IMO.